_data/sites.json

```diff
@@ -8918,8 +8918,9 @@
 
     {
         "name": "PhishTank",
-        "url": "https://www.phishtank.com",
-        "difficulty": "impossible",
+        "url": "https://privacyrequest.cisco.com",
+        "difficulty": "hard",
+        "notes": "Select \"Phishtank\" under \"Account(s) or Service(s) of interest\"",
```

Comment:
Then what? Is there a dropdown stating that the reason is deletion of the account?

Reply:
That goes in request details (added to note)

Reply:
Thanks